skorch/net.py

```diff
@@ -456,6 +458,30 @@ def initialize_module(self):
         self.module_ = module.to(self.device)
         return self
 
+    def register_runtime_param(self, param_patterns, fn):
+        self.runtime_params_ = getattr(self, 'runtime_params_', {})
+        if not isinstance(param_patterns, list):
```

Comment:
Since `register_runtime_param` is an internal api, it could be good to force `param_patterns` to just always be a list. (Reduces the lines of code)